src/client/data/shape.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
```

Comment:
`shape` seems irrelevant to `data`. Create a new folder name `type` or `shape` would be suitable.

```diff
  ├── client
  │   ├── App.js
  │   ├── action
  │   ├── assets
+ │   ├── types
  │   ├── data
  │   ├── index.js
  │   └── site
```